pkgs/applications/audio/sublime-music/default.nix

```diff
@@ -51,10 +52,20 @@ python3Packages.buildPythonApplication rec {
   # https://github.com/NixOS/nixpkgs/issues/56943
   strictDeps = false;
 
+  postInstall = ''
+    install -Dm444 sublime-music.desktop      -t $out/share/applications
```

Comment:
Would be nice if upstream install script could install these but does not look like poetry supports that at the moment: https://github.com/python-poetry/poetry/issues/890

Reply:
I don't want to hard-code the installation of the `.desktop` and icon files into the upstream package. Even if Poetry supported it, I think it would cause issues if users do a `--user` install.

Reply:
They would be installed to `$prefix` so for user install, it would be under `~/.local`.

Reply:
Hmm... I haven't really seen any examples of a Python package actually respecting that. Happy to integrate something along those lines if you can point me to it.

Reply:
I believe that if you make the paths relative like

https://github.com/MeanEYE/Sunflower/blob/3c45a4cbd28b785d3d49f3fc0dca5e8227ee5d29/setup.py#L29-L33

it will just install it to whatever path user specifies with `--prefix`. It is setuptools specific, though.

Most Python projects I have seen migrated to Meson. For example: https://gitlab.gnome.org/GNOME/gnome-music

Reply:
Looks like quodlibet uses gdist and poetry though: https://github.com/quodlibet/quodlibet/blob/e6b2c65eada1bc9625327ab560029215f74fc67b/setup.py#L94-L121

Reply:
@jtojnar I opened this issue on the Sublime Music repo: https://gitlab.com/sublime-music/sublime-music/-/issues/282

I just migrated Sublime Music from setuptools+Pipenv to poetry, and I really don't want to have to make another major infrastructure change anytime soon. I can try to figure out how to make Poetry install the files in the correct locations, but that's fairly involved and is going to be low priority considering there are still many user-visible bugs and missing features in Sublime Music that need to be solved before I try and improve this process.

Considering that the approach I've taken in this PR works, I am inclined to ask to have this merged and revisit this process later once I find the time (or once somebody comes along and contributes a fix for this to the Sublime Music project :joy:).

Reply:
Sure, it can be fixed later.